client/src/Components/MyApplications/style.css

```
@@ -0,0 +1,8 @@
+.my-applications__title {
```

```suggestion
.my-applications__title {
    color: rgba(51, 51, 51, 0.8);
    font-size: 32px;
    padding-bottom: 15px;
    margin-bottom: 1.3rem;
    border-bottom: 2px solid rgba(51, 51, 51, 0.8);
    width: fit-content; 
    margin-left: 7px;
}
```
 to be same as me and fatma done.

---

ok, I'll do it, but please rely on figma prototype.